python/dcr_template.py

```diff
@@ -287,67 +298,71 @@ def __init__(self, repository=".", obsid_range=None, band_name='g', wavelength_s
         self.use_psf = use_psf
         self.use_fft = use_fft
         self.debug = debug_mode
-        self._build_regularization()
+        self.regularize = self._build_regularization(x_size=self.kernel_size,
+                                                     y_size=self.kernel_size, n_step=self.n_step)
         self._calc_psf_model()
 
-    def _build_regularization(self):
+    @staticmethod
+    def _build_regularization(x_size=None, y_size=None, n_step=None):
         """
         Regularization adapted from Nate Lust's DCR Demo iPython notebook.
         Calculate a difference matrix for regularization as if each wavelength were a pixel, then scale
         the difference matrix to the size of the number of pixels times number of wavelengths
         """
-        # baseReg = _difference(self.kernel_size)
-        # reg_pix = np.zeros((self.n_step * self.kernel_size, self.n_step * self.kernel_size))
+        # baseReg = _difference(kernel_size)
```

Comment:
Agreed. I have gone through and removed all unneeded commented-out code, and added comments to still useful test code.